Documentation/Guides/SSL Pinning.md

```diff
@@ -0,0 +1,36 @@
+### SSL Pinning
+
+SSL Pinning is the practice of limiting the set of server certificates your app trusts when making network requests. This occurs in addition to the trust verification the operating system performs by default. This, in combination with HTTPS, allows you to better preserve the privacy and integrity of the information you are transmitting back and worth to a backend server.
```

Comment:
should say "back and forth", but says "back and worth"

Reply:
🤦 Fixed